### PR TITLE
Replace pyswsssdk with sonic-py-common

### PR DESCRIPTION
**Pull request by liuh-80**
**- What I did**
Replace sonic-py-swsssdk with sonic-py-common.

**- How I did it**
Update all sonic-py-swsssdk to use sonic-py-common.

**- How to verify it**
Pass all UT.
Pass all E2E test.

**- Description for the changelog**
Replace sonic-py-swsssdk with sonic-py-common.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ steps:
 - script: |
     set -xe
     sudo pip3 install swsssdk-2.0.1-py3-none-any.whl
```
Comment (liuh-80) [inline azure-pipelines.yml:48]: Need keep swsssdk because UT still using it.

File: src/lldp_syncd/__main__.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import logging.handlers
 import sys
 
-import swsssdk.util
+import sonic_py_common.util
```
Comment (liuh-80) [inline src/lldp_syncd/__main__.py:4]: Please help do a force merge because code change in__main__.py not covered by UT.

File: setup.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 from setuptools import setup, find_packages
 
 dependencies = [
-    'swsssdk>=1.3.0',
```
Comment (liuh-80) [inline setup.py:4]: Remove dependency, keep this line will cause following error in build-image:
2022-07-18T07:28:33.7302099Z Processing ./sonic_d-2.0.0-py3-none-any.whl
2022-07-18T07:28:33.7302851Z [91mERROR: Could not find a version that satisfies the requirement swsssdk>=1.3.0 (from sonic-d) (from versions: none)

File: setup.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
...
 ]
 
 test_deps = [
```
Comment (qiluo-msft) [inline setup.py:6]: >test_deps [](http://example.com/codeflow?start=0&length=9)

Do you need adding `swsssdk` into `test_deps`?

Comment (liuh-80) [reply setup.py:6]: Fixed.

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
I will override the coverage checker since the missing lines are not covered by unit test, and could be verified by E2E test.

**Pull request merged**